src/reference/glossary.rst

```diff
@@ -30,15 +36,11 @@ Glossary
 
       A :term:`workflow` maintained by the Nextstrain team.
 
-   workflow repository
-      also *pathogen workflow repository*
-
-      A version-controlled folder containing all files necessary to run a :term:`workflow`.
-
    build
       also *Nextstrain build*
 
-      *(noun)* A sequence of commands, parameters and input files which work together to reproducibly execute bioinformatic analyses and generate a :term:`dataset` for visualization with :term:`Auspice`.
+      *(noun)* A sequence of commands, parameters and input files which work together to reproducibly generate outputs.
+      Phylogenetic builds execute bioinformatic analyses and generate a :term:`dataset` for visualization with :term:`Auspice`.
 
```

Comment:
Do you see ingest/nextclade as other types of builds, analagous to the types of workflows?

Reply:
Yes? To me, the terms `build` and `workflow` really start to blend together here...If we define the 3 different types of workflows [as you suggested](https://github.com/nextstrain/docs.nextstrain.org/pull/191#discussion_r1511690095), then I think we can remove the specific definitions for types of builds.

Reply:
Ah, I see! Yes, ingest/nextclade both can have this distinction as well. 

For nextclade, the mpox nextclade workflow has [3 different builds](https://github.com/nextstrain/mpox/blob/e439235ff1c1d66e7285b774e9536e2896d9cd2f/nextclade/Snakefile#L6) that produces a different [mpox Nextclade dataset](https://github.com/nextstrain/nextclade_data/tree/master/data/nextstrain/mpox).

For ingest, this might not be as clear. For example, the [dengue ingest workflow](https://github.com/nextstrain/dengue/tree/main/ingest#usage) produces `all`, `denv_1`, `denv_2`, `denv_3`, `denv_4` data files, where denv 1-4 are different serotypes that are subsets of the full data. I think each serotype can be considered a different build?

Reply:
Got it! In that case, it might also make sense to generalize the term **dataset** to be the output of the build process:

1. Phylogenetic dataset = the output of an Augur build (to be used by Auspice)
2. Nextclade dataset = the output of a Nextclade build
3. Ingest dataset = the output of an ingest build (e.g. for a specific dengue serotype)

Reply:
I think generalizing the term `dataset` is a good direction, but I'm also hesitant because it's so closely tied to Auspice JSONs in all of our docs and code. I guess we can add a special note to the term `dataset`:

```
Unless otherwise specified, a "dataset" usually refers to a set of Auspice JSONs
```

Reply:
👍 special note would be good. Similar to this note from [another suggestion](https://github.com/nextstrain/docs.nextstrain.org/pull/191#discussion_r1511690095):

> Often considered the primary workflow in a pathogen repository (e.g. "the Zika workflow" typically means "the phylogenetic workflow in the Zika pathogen repository").

Reply:
Updated in 7a7b50c1b422be2f06b031844bd31e388c8c78fe

Reply:
The addition of individual terms for **workflow** and **dataset** seems incomplete without doing the same for **build**.

Suggestion: add individual terms for *phylogenetic/ingest/Nextclade build* as described in https://github.com/nextstrain/docs.nextstrain.org/pull/191#discussion_r1511939744

Reply:
Hmm, I didn't add individual terms for build because I had thought the [current definition](https://nextstrain--191.org.readthedocs.build/en/191/reference/glossary.html#term-build) applies across all three.

I'm having trouble differentiating the phylogenetic/ingest/Nextclade build terms. 
This seems a bit overkill to me, but you probably have a better suggestion? 

```rst

phylogenetic build 

    (noun) A sequence of commands, parameters and input files which work together to 
    reproducibly generate a phylogenetic dataset. Phylogenetic builds are often have distinct 
    input files and subsampling schemes for slightly different bioinformatic analyses.

ingest build
    
    (noun) A sequence of commands, parameters and input files which work together to 
    reproducibly generate an ingest dataset. Ingest builds often have distinct upstream data 
    sources and groupings by pathogen subtypes or genes.

nextclade build

    (noun) A sequence of commands, parameters and input files which work together to 
    reproducibly generate a Nextclade dataset. Nextclade builds often have distinct 
    reference sequences and subsampling schemes to be used for different scopes of Nextclade analyses.
```

Reply:
Oh you're right, it does seem overkill. Maybe better to stick with a single entry with more specific aliases:

> **build**
>
>  also *Nextstrain build*, *phylogenetic build*, *ingest build*, *Nextclade build*

Reply:
Updated in https://github.com/nextstrain/docs.nextstrain.org/pull/191/commits/63b716bd1ac64d18215fc42ce85ae0b6ae8a23bc